symphony-bdk-core/src/main/java/com/symphony/bdk/core/auth/impl/AuthenticationRetry.java

```diff
@@ -78,11 +83,13 @@ public T executeAndRetry(String name, SupplierWithApiException<T> supplier, Stri
       return retry.execute();
     } catch (ApiException e) {
       if (e.isUnauthorized()) {
+        log.error(unauthorizedErrorMessage);
```

Comment:
Don't we want to let the caller decides if it should log or not the error?

Reply:
I added the log.error here for the same reason as above https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/pull/421#discussion_r567641953 . But if the reason is not valid I can remove it.

Reply:
removing this as well